leftwm/src/config/keybind.rs

```diff
@@ -28,11 +28,36 @@ impl Keybind {
         &self,
         config: &Config,
     ) -> Result<lefthk_core::config::Keybind> {
+        fn is_valid_scratchpad_name(config: &Config, scratchpad_name: &str) -> bool {
+            config
+                .scratchpad
+                .as_ref()
+                .and_then(|scratchpads| scratchpads.iter().find(|s| s.name == scratchpad_name))
+                .is_some()
+        }
+
```

Comment:
I'd move this function out because the logic of this function is only relevant for two base commands and not for the whole function `try_convert_to_lefthk_keybind`.

Reply:
Where would you suggest to put this function?

Reply:
This is a hard question.... I'd suggest to put it completely at the bottom of the file. :thinking: